src/Eto/Eto.csproj

```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.0</TargetFramework>
+	<GenerateDocumentationFile>true</GenerateDocumentationFile>
+  </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
+    <DefineConstants>TRACE;DEBUG;PCL</DefineConstants>
+    <OutputPath>..\..\artifacts\core\Debug\</OutputPath>
```

Comment:
You can move this to main `<PropertyGroup>`
`<OutputPath>..\..\artifacts\core\$(Configuration)\</OutputPath>`

Reply:
I tried putting this outside all projects in the shared props file, but I think it's because I was trying to be too aggressive.  I think you're right in that for sdk-style csproj's I can do this.